doc/source/features.csv

```diff
@@ -0,0 +1,118 @@
+"  `Allow replacing master cross-signing key without... <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#allow-replacing-master-cross-signing-key-without-user-interactive-auth>`_",
+"  `User devices <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#user-devices>`_","---"
+"    -- `List all devices <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#list-all-devices>`_",:option:`synadm user prune-devices --list-only`
+"    -- `Create a device <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#create-a-device>`_",
+"    -- `Delete multiple devices <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#delete-multiple-devices>`_",:option:`synadm user prune-devices -d` :option:`synadm user prune-devices -s`
+"    -- `Show a device <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#show-a-device>`_",:option:`synadm user prune-devices --list-only` (-i)
+"    -- `Update a device <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#update-a-device>`_",
+"    -- `Delete a device <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#delete-a-device>`_",:option:`synadm user prune-devices -i`
+"  `List all pushers <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#list-all-pushers>`_",
+"  `Controlling whether a user is shadow-banned <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#controlling-whether-a-user-is-shadow-banned>`_",:option:`synadm user shadow-ban USER_ID` :option:`synadm user shadow-ban --unban`
+"  `Override ratelimiting for users <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#override-ratelimiting-for-users>`_","---"
+"    -- `Get status of ratelimit <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#get-status-of-ratelimit>`_",
+"    -- `Set ratelimit <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#set-ratelimit>`_",
+"    -- `Delete ratelimit <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#delete-ratelimit>`_",
+"  `Check username availability <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#check-username-availability>`_",
+"  `Find a user based on their ID in an auth provider <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#find-a-user-based-on-their-id-in-an-auth-provider>`_",:option:`synadm user 3pid ADDRESS`
+"  `Find a user based on their Third Party ID <https://element-hq.github.io/synapse/develop/admin_api/user_admin_api.html#find-a-user-based-on-their-third-party-id-threepid-or-3pid>`_",:option:`synadm user auth-provider EXTERNAL_USER_ID`
+"`Version API <https://element-hq.github.io/synapse/develop/admin_api/version_api.html>`_",:option:`synadm --version`
```

Comment:
Yes that's confusing. --version is kind of a standard in cli tools though, so ok. synadm version is an admin api command, so actually ok too. Not sure how this could be improved.